README.md

```diff
@@ -1 +1,49 @@
 ## Welcome to the Open Horizon Services project!
+
+This project was created to house edge micro-services that can be easily built, tested, and published to any open horizon management hub.
+
+## Learn more about Open Horizon 
+**Open Horizon documentation repository coming soon!** For the time being, you can learn more about [Open Horizon here](https://www.ibm.com/support/knowledgecenter/SSFKVV_4.2/kc_welcome_containers.html).
+
+## Management Hub Installation
+Before you can publish and use any of the services in this repository, you must first deploy your own Horizon Management Hub. This can be done with one simple command using the `deploy-mgmt-hub.sh` script located in the [devops repository](https://github.com/open-horizon/devops/tree/master/mgmt-hub#horizon-management-hub). This will give you with a management hub with several services, policies and patterns published in the exchange. 
+
+## Register an Edge Node with your Mangement Hub
+In order to deploy a service to an edge node it must first be registered with a management hub. The `agent-install.sh` script is a fast and easy way to register an edge node with a management hub, more information can be found in the [open-horizon/anax](https://github.com/open-horizon/anax/tree/master/agent-install#edge-node-agent-install) repository. Edge nodes can be either a device or a cluster. Open Horizon edge cluster capability helps you manage and deploy workloads from a management hub cluster to remote instances of OpenShift® Container Platform or other Kubernetes-based clusters. 
```

Comment:
In the first sentence, it's unclear what needs to be registered with the management hub. While the second sentence makes it clear, I think the first sentence should still be something like: "The edge node must first be registered with a management hub before you can deploy a service to that node."